src/Api/Vault/AuthorizationHandlers/Collections/BulkCollectionAuthorizationHandler.cs

```diff
@@ -96,17 +101,22 @@ public class BulkCollectionAuthorizationHandler : BulkAuthorizationHandler<BulkC
     private async Task CanCreateAsync(AuthorizationHandlerContext context, IAuthorizationRequirement requirement,
         CurrentContextOrganization? org)
     {
-        // If the limit collection management setting is disabled, allow any user to create collections
-        // Otherwise, Owners, Admins, and users with CreateNewCollections permission can always create collections
+        // Owners, Admins, and users with CreateNewCollections permission can always create collections
```

Comment:
This kinda makes the logic easier to understand as well! It separates out the user role vs. the organization setting.